### PR TITLE
Switch the  CLI to using yargs

### PR DESCRIPTION
**Pull request by istarkov**
## Description

Switch on yargs in cli parser.

Shows help on specific commands too

image

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: packages/cli/src/commands/yargs-types.ts [hunk 1]
```diff
@@ -0,0 +1,65 @@
+/*
```
Comment (kof) [inline packages/cli/src/commands/yargs-types.ts:1]: that's weird, why is this not in the package typings ... I guess rhetorical question

Comment (istarkov) [reply packages/cli/src/commands/yargs-types.ts:1]: In most cases this types are not needed.
Like
```
cmd.command(
  ['deploy [script]'],
  '🆙 Deploy your Worker for Platforms to Cloudflare.',
  // builder
  (yargs: Argv) =>
    yargs
      .positional('script', {
        describe: 'The path to an entry point for your worker',
        type: 'string',
        requiresArg: true,
        demandOption: true,
      })
      .option('name', {
        describe: 'Name of the worker',
        type: 'boolean',
        demandOption: true,
      }),
  // handler
  async (options) => {
    console.log(options.name);
  }
);
```
will be extracted automatically.
Cloudflare made 2 things:
1. Allow  command builder to be in the other file
2. Make handler more strict (removing [key:string]: unknown) from type so `options.anyRandomName` start giving an error (in original yargs it is not an error just `unknown`)

Comment (kof) [reply packages/cli/src/commands/yargs-types.ts:1]: thats intetresting info that would be nice in there

**Review by kof — APPROVED** [review]

**Review by JayaKrishnaNamburu — APPROVED** [review]
Looks good 💯

**Review by TrySound — APPROVED** [review]

**Pull request merged**